Setup/UpgradeData.php

```diff
@@ -30,6 +84,13 @@ public function upgrade(
             $this->version_2_1_7($context, $setup);
         }
 
+        if (version_compare($context->getVersion(), "2.3.2", "<=")) {
```

Comment:
we added two version updates here. Starting version is 2.3.2. Check here should be <. And should be done in one version up both upgrades for version 2.3.3.
Module version should be upgraded in module.xml not to 2.3.4 but to 2.3.3

Reply:
Thanks - done